executor/oci/spec_unix.go

```diff
@@ -139,3 +146,12 @@ func getTracingSocketMount(socket string) specs.Mount {
 func getTracingSocket() string {
 	return fmt.Sprintf("unix://%s", tracingSocketPath)
 }
+
+func cgroupNamespaceSupported() bool {
```

Comment:
Moby, containerd, etc. disables cgroup namespace on cgroup v1 regardless to kernel version/config.

Reply:
Where is this done? In Runc code I see it checks ns existence for both cgroup versions.

Reply:
https://github.com/moby/moby/blob/c57097bcd461fcbc31f52b249c336f5c2e9732e8/daemon/daemon_unix.go#L345-L347

Reply:
Iiuc this means privileged containers get host cgroups on v1? Is this what you mean and we should replicate it for `--security=insecure`. I don't quite see how it makes sense though to have different behaviour for privileged containers based on group versions.

Reply:
Also see these consts in moby https://github.com/moby/moby/blob/462d6ef826861fad021fb565c0481fb61d2db6bc/daemon/config/config_linux.go#L23-L27 now used in https://github.com/moby/moby/blob/462d6ef826861fad021fb565c0481fb61d2db6bc/daemon/config/config_linux.go#L189-L193. So if not v2 then always set `host` mode.